content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Add custom methods to your models'
+metaTitle: 'Prisma Client extensions: model (Preview)'
+metaDescription: 'Extend the functionality of Prisma Client'
+tocDepth: 3
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+This page is in draft.
+
+</Admonition>
+
+From version **TBD**, you can use `model` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) to add custom methods to your models.
+
+Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an _extended client_. An extended client is a forked variant of the standard Prisma Client with an extension applied. Use the `model` extension type to add methods to a specific model in your schema or to all models in your schema.
+
+Possible uses for `model` include the following:
+
+- New operations to operate alongside existing Prisma Client operations, such as `findMany`
+- Encapsulated business logic
+- Repetitive operations
+- Model-specific utilities
+
+</TopBlock>
+
+## Enable the preview feature
```

Comment:
Same question for this section https://github.com/prisma/docs/pull/4004/files#r1016855600

Reply:
To make this page self-contained where possible, I've repeated this section. There is a fine balance between repeating too much material and not providing enough in that context, but we must always assume that a user might drop into the page (or a self-contained section of a longer page like the Client reference) from Google.